test/index.js

```diff
@@ -20,8 +20,10 @@ series([
   require('./basic')
   require('./multitarget')
 
-  if (process.platform !== 'win32') {
-    // Perform additional tests specific to building for OS X
+  // Perform additional tests specific to OS
+  if (process.platform === 'darwin') {
     require('./mac')
+  } else if (process.platform === 'win32') {
+    require('./win32')
```

Comment:
Is it possible to run these tests via wine (particularly on Linux, so we can integrate them into the repository's continuous integration)?